addons/web/static/src/scss/bootstrap_review_frontend.scss

```diff
@@ -248,10 +255,20 @@ $-color-for-gray-200-bg: adjust-color-to-background($body-color, $gray-200);
 }
 
 // Dropdown
-.dropdown .dropdown-menu {
+.dropdown .dropdown-menu,
+.btn-group .dropdown-menu {
     .text-muted {
         color: adjust-color-to-background($text-muted, $dropdown-bg, mute-color($color-contrast-light), mute-color($color-contrast-dark)) !important;
     }
+
+    // Since we define the $dropdown-link-active-bg variable, we need to
+    // redefine the .active state of the dropdown items to differentiate
+    // an active class from the active state. This allows a discreet effect for
+    // the active state while keeping an active navigation element prominent.
+    .dropdown-item.active {
+        --dropdown-link-active-bg: #{$component-active-bg};
+        --dropdown-link-active-color: #{$component-active-color};
+    }
```

Comment:
It's strange because after your changes, the active link for the dropdown in the search input on the `/my/tasks` page of the portal no longer has a primary background.
![image](https://github.com/user-attachments/assets/9dd943c0-a019-4821-b691-af4720d16778)
could you check please 🙏🏼

Reply:
Good catch! Selector issue, it's in neither a `.btn-group` nor a `.dropdown` this `.dropdown-menu` is in a`.input-group`, at this stage I think we're better off removing specificity and targeting `.dropdown-menu` directly.